lib/purchases/class/product.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:myecl/purchases/class/ticket_generator.dart';
+
+class Product {
+  Product({
+    required this.id,
+    required this.nameFR,
+    required this.nameEN,
+    required this.descriptionFR,
+    required this.descriptionEN,
+    required this.ticketGenerators,
+  });
+
+  late final String id;
+  late final String nameFR;
```

Comment:
Is the translation really used ? Could we use a map instead of creating specific cases variable ?

Reply:
Translation was implemented for the website, only two languages are used French and English